Simplified/NYPLKeychainManager.swift

```diff
@@ -160,38 +162,67 @@ import Foundation
       ]
       let status = SecItemAdd(addQuery as CFDictionary, nil)
       if status != errSecSuccess && status != errSecDuplicateItem {
-        // This is unexpected
-        var errMsg = ""
-        if #available(iOS 11.3, *) {
-          errMsg = (SecCopyErrorMessageString(status, nil) as String?) ?? ""
-        }
-        if errMsg.isEmpty {
-          switch status {
-          case errSecUnimplemented:
-            errMsg = "errSecUnimplemented"
-          case errSecDiskFull:
-            errMsg = "errSecDiskFull"
-          case errSecIO:
-            errMsg = "errSecIO"
-          case errSecOpWr:
-            errMsg = "errSecOpWr"
-          case errSecParam:
-            errMsg = "errSecParam"
-          case errSecWrPerm:
-            errMsg = "errSecWrPerm"
-          case errSecAllocate:
-            errMsg = "errSecAllocate"
-          case errSecUserCanceled:
-            errMsg = "errSecUserCanceled"
-          case errSecBadReq:
-            errMsg = "errSecBadReq"
-          default:
-            errMsg = "Unknown OSStatus: \(status)"
-          }
-        }
+        logKeychainError(for: status, with: "FeedbookKeyManagement Error:")
+      }
+    }
+  }
+    
+  private class func manageFeedbookDrmPrivateKey() {
+    for vendor in AudioBookVendors.allCases {
+      guard let privateKeyString = NYPLSecrets.drmCertificate(forVendor: vendor),
+        let privateKeyData = Data(base64Encoded: RSAUtils.stripPEMKeyHeader(privateKeyString)),
```

Comment:
can you add a comment on why we need to strip the PEM key header?

Reply:
added